### PR TITLE
Adot: Add OpenRTB macros resolution

### PR DESCRIPTION
**Pull request by ramyferjaniadot**
Hello,

This PR is about adding OpenRTB macros resolution in Adot adapter, as discussed here [https://github.com/prebid/prebid-server/issues/1804](https://github.com/prebid/prebid-server/issues/1804)

Sincerely,

### CODE REVIEW

File: adapters/adot/adot_test.go [hunk 1]
```diff
@@ -75,3 +75,11 @@ func TestMediaTypeForBid(t *testing.T) {
 		t.Errorf("the type is not the valid one. actual: %v, expected: %v", bidTypeNative, openrtb_ext.BidTypeVideo)
 	}
 }
+
+//Test macros resolution for adm and nurl
```
Comment (AlexBVolcy) [inline adapters/adot/adot_test.go:79]: Nitpick, I would say the comments above this test function, along with the others in this file aren't necessary. The name of the test function itself describes what the test is doing well enough!

Comment (ramyferjaniadot) [reply adapters/adot/adot_test.go]: They were already there so i kept the same norm, but i understand your point.
I removed them.

**Review by VeronikaSolovei9 — APPROVED** [review]
LGTM

**Review by AlexBVolcy — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**